README.md

```diff
@@ -1 +1,43 @@
-# To-do-List-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do List
+
+## Description 
+> This project has been assigned to us by microverse.
+> To-do-list is an oline website where it can be planned and managed time by adding a list of task. tasks can be removed , added, and edited.
+
+## Built With
+- Languages: HTML5, CSS5, JavaScript
+- VCS: GitBash
+- Package Manager: Node, NPM
+-Following github flows standards.
+
+## Live demo 
+[click here](https://tarikbouari.github.io/Awesome-books/)
```

Comment:
Optional

I believe this is not the live link for this project, please fix it or remove for a professional Readme file